jetty-websocket/websocket-jetty-server/src/main/java/org/eclipse/jetty/websocket/server/JettyWebSocketServerContainer.java

```diff
@@ -267,9 +267,22 @@ public void setAutoFragment(boolean autoFragment)
     @Override
     public CompletableFuture<Void> shutdown()
     {
-        LifeCycle.stop(sessionTracker);
         CompletableFuture<Void> shutdown = new CompletableFuture<>();
-        shutdown.complete(null);
+        new Thread(() ->
```

Comment:
Same comments as above.... looking like we need a utility class to do this threadful stopping.